tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -1,22 +1,34 @@
-import { shallow } from 'enzyme';
 import React from 'react';
 
-import { AddonMetaBase } from 'amo/components/AddonMeta';
-import { fakeAddon } from 'tests/unit/amo/helpers';
-import { getFakeI18nInst } from 'tests/unit/helpers';
+import AddonMeta, { AddonMetaBase } from 'amo/components/AddonMeta';
+import Link from 'amo/components/Link';
+import {
+  dispatchSignInActions,
+  dispatchClientMetadata,
+  fakeAddon,
+} from 'tests/unit/amo/helpers';
+import { getFakeI18nInst, shallowUntilTarget } from 'tests/unit/helpers';
 import LoadingText from 'ui/components/LoadingText';
 import Rating from 'ui/components/Rating';
 
-function render({ ...customProps } = {}) {
-  const props = {
-    addon: fakeAddon,
-    i18n: getFakeI18nInst(),
-    ...customProps,
-  };
-  return shallow(<AddonMetaBase {...props} />);
-}
 
 describe('<AddonMeta>', () => {
+  function render({
+    addon = fakeAddon,
```

Comment:
We should get into the habit of doing `addon = createInternalAddon(fakeAddon)` because the internal addon object is what this component will receive. You can import it from https://github.com/mozilla/addons-frontend/blob/master/src/core/reducers/addons.js#L83

Reply:
Should `fakeAddon` just get wrapped in `createInternalAddon()`? I noticed you doing that in another test file a lot and it seems like boilerplate. Maybe we want a helper like `fakeAddon({ ...extraProps })` instead of the object we manipulate now.